src/backend/columnar/columnar_storage.c

```diff
@@ -86,7 +86,7 @@ typedef struct PhysicalAddr
 #define COLUMNAR_INVALID_STRIPE_ID 0
 #define COLUMNAR_FIRST_STRIPE_ID 1
 #define COLUMNAR_INVALID_ROW_NUMBER 0
-#define COLUMNAR_FIRST_ROW_NUMBER 1
+#define COLUMNAR_FIRST_ROW_NUMBER 0
```

Comment:
since all my changes were based on `COLUMNAR_FIRST_ROW_NUMBER` being equal to `0`, I changed this and the regression tests outputs

Reply:
reverted that change